testsuite/MDAnalysisTests/__init__.py

```diff
@@ -174,4 +175,14 @@ def executable_not_found(*args):
                 break
         return not found
 
-    return lambda: executable_not_found(*args)
+    return lambda: executable_not_found_runtime(*args)
+
+def module_not_found(module):
+    def module_not_found_runtime(module):
+        try:
+            importlib.import_module(module)
+        except ImportError:
+            return True
+        else:
+            return False
+    return lambda: module_not_found_runtime(module)
```

Comment:
Why are we using this weird wrapping thing?  Does this make it only attempt the import as the test is started rather than before?